src/main/java/nl/tudelft/ti2206/group9/gui/scene/SettingsScene.java

```diff
@@ -196,9 +215,35 @@ private static Slider createVolumeSlider(
         final BackgroundFill fill = new BackgroundFill(color, corner, inset);
         final Background sliderBack = new Background(fill);
         slider.setBackground(sliderBack);
+        setSliderFunction(slider, sliderType);
         return slider;
     }
 
+    /**
+     * Sets the volume of the the sounds that are used in the application.
+     * The audioplayer volumes of the 'non-GameScenes' are already set below.
+     * The others are set in the GameScene (SoundEffectObserver).
+     * @param slider the given slider to set the function of.
+     * @param type the given type of slider to set the volume of.
+     */
+    protected static void setSliderFunction(final Slider slider,
+            final SType type) {
+        slider.setOnMouseReleased(event -> {
+            if (type == SType.VOLUME_SOUNDTRACK) {
+                State.setSoundtrackVolume(soundtrackVolumeSlider.
+                        getValue() / VOLUME_CONVERTER);
+                MainMenuScene.getAudioPlayer().
+                        setVolume(State.getSoundtrackVolume());
+            } else {
+                State.setSoundEffectVolume(soundEffectVolumeSlider.
+                        getValue() / VOLUME_CONVERTER);
+                AbstractMenuScene.getButtonSoundEffectPlayer().
+                        setVolume(State.getSoundEffectVolume());
+                AbstractInfoPopup.getButtonSoundEffectPlayer().
```

Comment:
Why would you set the volume here if you also set it just before playing a SoundEffect? If you remove these three lines, you can also get rid of the getters in InfoPopup and MenuScene.

<review, interface